jupyter-book/spatial/imputation.yml

```diff
@@ -0,0 +1,11 @@
+name: imputation
+channels:
+  - defaults
+  - conda-forge
+dependencies:
+- python>=3.9
+- pip=20.2.2
+- pip:
+  - scanpy[leiden]
```

Comment:
Can we please pin channels for all dependencies here? Also, please always use conda whereever possible and then pip for the rest

-> `conda-forge::scanpy` etc